content/guides/installing/making_haiku_usb_stick.html

```diff
@@ -69,15 +69,15 @@
 
 <h2>Using ImageWriter for Windows</h2>
 
-<div class="alert alert-info">Note that ImageWriter for Windows is considered experimental (alpha) software by it's own author. Be sure to read the README.txt file included with the software.</div> 
+<div class="alert alert-info">Note that ImageWriter for Windows is considered beta software by it's own author. Be sure to read the README.txt file included with the software.</div> 
```

Comment:
Er ... since  you changed the below to use Win32DiskImager shouldn't you change this as well as the h2 also?